server/src/infra/ldap_handler.rs

```diff
@@ -204,6 +210,333 @@ fn root_dse_response(base_dn: &str) -> LdapOp {
                 atype: "isGlobalCatalogReady".to_string(),
                 vals: vec![b"false".to_vec()],
             },
+            LdapPartialAttribute {
+                atype: "subschemaSubentry".to_string(),
+                vals: vec![b"cn=Subschema".to_vec()],
+            },
+        ],
+    })
+}
+
+pub struct ObjectClassList(Vec<LdapObjectClass>);
+
+impl ObjectClassList {
+    fn format_for_schema(&self) -> String {
```

Comment:
format_for_ldap_schema_description maybe?
There was another instance of that function, let's also align.

For both, could you point to the RFC paragraph that describes the format?